### DISCUSSION TITLE
Embedding "Breadcrumbs Down" View in Note - Like a Dataview Query

### DISCUSSION THREAD

**Discussion by nschutz** [SkepticMystic/breadcrumbs#412] [Q&A] [answered]
The "Breadcrumbs Down" view is perfect, and I'd love to have it embedded in certain project notes so I can easily view the "index" of the project without using the "copy local index" command and pasting. 

I currently use a dataview query which includes the `where` query of, for example, `contains(parent, [[🔬 Journal App]` but this won't include implied relationships or granchildren notes the way the "Breadcrumbs Down" view does.

Any known ways to do this? Not a big deal but it'd be nice, thanks!

**Comment by SkepticMystic** [comment, marked as answer]
Hey @nschutz :)
There are a couple of options here.
The goal is to show the Breadcrumbs down view in a specific note. I see two ways you could acheive this:

1. Use BC [Codeblocks](https://breadcrumbs-wiki.onrender.com/docs/Codeblocks) which let you do just that!
2. Use the BC [API](https://breadcrumbs-wiki.onrender.com/docs/API) which lets you access the BC index programmatically (dataviewjs included). See the [docs](https://skepticmystic.github.io/breadcrumbs/interfaces/BCAPII.html) for more info

Unfortunately, I don't think there is an easy way to make Dataview see what BC sees. That would require a change to DV's code. But the Codeblocks _should_ do what you're looking for, and the API is the next best thing to integrate BC with DV

Let me know if this helps!

**Reply by nschutz** [reply]
Wow the codeblocks query was **exactly** what I was looking for, thank you so much!

So awesome, thanks for such a quick response too

**Comment by selimrbd** [comment]
Hey @SkepticMystic 

Is there any way to order the results (say alphabetically by file name ?)

**Reply by SkepticMystic** [reply]
Hey :)
Not currently, no.. but there are definitely issues open for this feature request! It's quite a challenging one

**Comment by michaelpporter** [comment]
Closing — embedding a "Down" view in a note is now first-class via the `breadcrumbs` codeblock. Reopen if that doesn't cover your case.
